extensions/wikia/Search/classes/Services/ESFandomSearchService.php

```diff
@@ -70,13 +72,16 @@ protected function consumeResponse( $response ) {
 		$results = [];
 
 		foreach ( $response as $item ) {
-			if ( isset( $item['_source'] ) ) {
+			if ( isset( $item['_source'] ) && isset( $item['_source']['title'] )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "_source" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "title" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)